client/src/hooks/graphql/use-with-review-answer-state.tsx

```diff
@@ -152,15 +152,11 @@ export function useWithReviewAnswerState(
     }
   }, [trainError]);
 
-  function recordAnswers(
-    status: Status,
-    subject: string | undefined,
-    category: string | undefined
-  ) {
+  function recordAnswers(status: Status, subject: string, category: string) {
     navigate(
       `/record?back=${encodeURI(
         `/?subject=${selectedSubject}`
-      )}&status=${status}&subject=${subject || ""}&category=${category || ""}`
+      )}&status=${status}&subject=${subject}&category=${category}`
```

Comment:
I asked Jaime to add a `urlBuild` helper function in another PR. Once that's in place, we should use it instead of this string-format stuff